### PR TITLE
UIU-1286: Fix the accumulation of overdue loans in CSV report.

### PR DESCRIPTION
**Pull request by SergiyVSergiyenko**
### Purpose
Fix the mechanism for the accumulation of overdue loans in the CSV report according to the [story](https://issues.folio.org/browse/UIU-1286).

### CODE REVIEW

File: src/Users.js [hunk 1]
```diff
@@ -127,6 +127,7 @@ class Users extends React.Component {
     loans: {
       type: 'okapi',
       records: 'loans',
+      accumulate: 'true',
```
Comment (VictorSoroka1) [inline src/Users.js]: This should be just `boolean`.

File: src/Users.js [hunk 2]
```diff
@@ -253,13 +254,23 @@ class Users extends React.Component {
     }
   }
 
+  fetchLoans = props => {
+    const {
+      reset,
+      GET,
+    } = props.mutator.loans;
+
+    reset();
```
Comment (VictorSoroka1) [inline src/Users.js]: I would also add some mechanism to prevent loans fetching when there is one already in progress.

File: src/Users.js [hunk 3]
```diff
@@ -253,13 +254,23 @@ class Users extends React.Component {
     }
   }
 
+  fetchLoans = props => {
```
Comment (VictorSoroka1) [inline src/Users.js]: I think the method is about to generate overdue loans report where fetching of loans is a kind of implementation detail. Please come up with another name.

Comment (SergiyVSergiyenko) [reply src/Users.js]: Thanks for the comment. What about 'loansToReport'?

Comment (YevheniiMaltsev) [reply src/Users.js]: @SergiyVSergiyenko usually the function should contains an action name - call, find, get, etc. So it looks like 'loansToReport' can be improved

**Pull request merged**